packages/create-gatsby/src/init-starter.ts

```diff
@@ -43,6 +40,9 @@ const checkForYarn = (): boolean => {
     execSync(`yarnpkg --version`, { stdio: `ignore` })
     return true
   } catch (e) {
+    reporter.info(
+      `Woops! Yarn doesn't seem be installed on your machine. You can install it on https://yarnpkg.com/getting-started/install. As a fallback, we will run the next steps with npm.`
```

Comment:
```suggestion
      `Woops! You have chosen "yarn" as your package manager, but it doesn't seem be installed on your machine. You can install it on https://yarnpkg.com/getting-started/install or change your preferred package manager with the command "gatsby options set pm npm". As a fallback, we will run the next steps with npm.`
```
Something like this might make it clearer